generators/app/about-us/about-us-ng-template-prompter.js

```diff
@@ -14,19 +14,19 @@ function askQuestions(generator) {
         {
             type: 'input',
             name: 'navElementKey',
-            message: 'What would you like the navigation element id to be? (e.g. aboutUs, about_us)',
+            message:
```

Comment:
please revert this.

Reply:
Please view the entire file. Github has underlined only a part, and it is not correct. There is the message, is not blank, the formatter has only add this in a new line because the string is too long.

Reply:
I know. I did look at the entire file.
I don't think I want this file formatted.
Kindly revert any changes resulting from a format or prettier changes